glib/src/variant.rs

```diff
@@ -187,6 +211,16 @@ impl Variant {
         T::from_variant(self)
     }
 
+    /// Tries to extract a value of type `T`.
+    pub fn try_get<T: FromVariant>(&self) -> Result<T, VariantTypeMismatchError> {
```

Comment:
```suggestion
    pub fn get<T: FromVariant>(&self) -> Result<T, VariantTypeMismatchError> {
```

IMHO (and remove the old `get()`). If someone needs the old behaviour then it's only a `.ok()` away.

Reply:
Ah hmm you're thinking to queue this for a new semver incompatible bump?  Is that already planned?  The 0.14 bump was *mostly* straightforward for us but it also reverberates up the stack - we had to do a chain of semver bumps for our code too.

I pushed a patch that is semver compatible for now that addresses the other comments.

Reply:
Ah I see https://github.com/gtk-rs/gtk-rs-core/commit/7c3814fc088f5ae876c7ebabd251f2d1071bf29c so that seems to answer that.

Reply:
Now that I try to change `get()` like you suggest it then raises the question of whether the `FromVariant` trait should also return a `Result<T, VariantTypeError>`.

Reply:
Keep it in a separate commit in the same PR and then we can backport the semver compatible changes to 0.14 and keep the breaking ones in master.

For `FromVariant`, I guess so. Ideally these traits should become closer to the ones for `Value`. But that's out of scope for this PR IMHO.